common/djangoapps/cme_registration_v2/cme_reg_form/models.py

```diff
@@ -0,0 +1,473 @@
+                          ('Centers - School of Medicine', 'Centers - School of Medicine'),
+                          ('Chemical and Systems Biology', 'Chemical and Systems Biology'),
+                          ('Comparative Medicine', 'Comparative Medicine'),
+                          ('Dermatology', 'Dermatology'),
+                          ('Developmental Biology', 'Developmental Biology'),
+                          ('Genetics Operations', 'Genetics Operations'),
+                          ('Health Research and Policy', 'Health Research and Policy'),
+                          ('Medicine', 'Medicine'),
+                          ('Microbiology and Immunology', 'Microbiology and Immunology'),
+                          ('Molecular and Cellular Physiology', 'Molecular and Cellular Physiology'),
+                          ('Neurobiology', 'Neurobiology'),
+                          ('Neurology', 'Neurology'),
+                          ('Neurosurgery', 'Neurosurgery'),
+                          ('Obstetrics & Gynecology', 'Obstetrics & Gynecology'),
+                          ('Ophthalmology', 'Ophthalmology'),
+                          ('Orthopaedic Surgery', 'Orthopaedic Surgery'),
+                          ('Otolaryngology/Head & Neck Surgery', 'Otolaryngology/Head & Neck Surgery'),
+                          ('Pathology', 'Pathology'),
+                          ('Pediatrics', 'Pediatrics'),
+                          ('Psychiatry and Behavioral Sciences', 'Psychiatry and Behavioral Sciences'),
+                          ('Radiation Oncology', 'Radiation Oncology'),
+                          ('Radiology', 'Radiology'),
+                          ('School of Medicine', 'School of Medicine'),
+                          ('SoM - Basic Science Pool', 'SoM - Basic Science Pool'),
+                          ('SoM - Bio-X/Clark', 'SoM - Bio-X/Clark'),
+                          ('SoM - Bioengineering', 'SoM - Bioengineering'),
+                          ('SoM - Clinical Science Pool', 'SoM - Clinical Science Pool'),
+                          ('SoM - Other Departments', 'SoM - Other Departments'),
+                          ('SoM Dean\'s Office Administrative Units', 'SoM Dean\'s Office Administrative Units'),
+                          ('SoM Non Cap Projects', 'SoM Non Cap Projects'),
+                          ('Stanford Cancer/Stem Cell Biology', 'Stanford Cancer/Stem Cell Biology'),
+                          ('Stanford Institutes of Medicine', 'Stanford Institutes of Medicine'),
+                          ('Structural Biology Department', 'Structural Biology Department'),
+                          ('Surgery', 'Surgery'),
+                          ('Urology', 'Urology'),
+                          ('Urology - Administration', 'Urology - Administration'),
+                          ('Urology - Divisions', 'Urology - Divisions'))
+    stanford_department = models.CharField(max_length=50, blank=True, choices=DEPARTMENT_CHOICES)
```

Comment:
i'm sure there is one, but what's the reason for being so specific about the charfield lengths? is it just consistency with the current schema?